addons/stock/models/stock_traceability.py

```diff
@@ -206,7 +206,7 @@ def get_pdf(self, line_data=[]):
             'base_url': base_url,
         }
 
-        body = self.env['ir.ui.view'].render_template(
+        body = self.env['ir.ui.view'].with_context(commit_assetsbundle=True).render_template(
```

Comment:
if you want the same behaviour as ir_actions_report's render_qweb_pdf you should guard this with_context with  if not config['test_enable']: